src/ray/raylet/node_manager.cc

```diff
@@ -424,7 +424,7 @@ NodeManager::NodeManager(instrumented_io_context &io_service,
             new rpc::RuntimeEnvAgentClient(ip_address, port, client_call_manager_));
       });
   worker_pool_.SetAgentManager(agent_manager_);
-
+  worker_pool_.Start();
```

Comment:
nit: just some concerns here. Now we need to call worker-pool start to start the worker. If we fail to call it, some function seems not working. (idle killing).
not big deal for now.